docs/tutorials/build/sync-table.md

```diff
@@ -0,0 +1,982 @@
+---
+title: Sync table
+description: Learn how to build a sync table of external data.
+icon: material/table-sync
+hide:
+- toc
+---
+
+# Learn to build a sync table
+
+<section class="tutorial-row" markdown>
```

Comment:
Nit, I found it a bit hard to read with the Goal box off to the side instead of above or below the text.

Reply:
Good callout, I may have been getting a bit too fancy. Simplified it here, and on other tutorials. Also standardized the intro language a little more.